backend/processing/bundle.bash

```diff
@@ -13,16 +13,17 @@ fi
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 work_dir="$(mktemp --directory)"
+asset_root='/asset-output'
 
 if [ -f "${script_dir}/${1}/requirements.txt" ]; then
     python -m venv "${work_dir}/.venv"
     # shellcheck source=/dev/null
     . "${work_dir}/.venv/bin/activate"
     python -m pip install --upgrade pip
 
-    pip install --requirement="${script_dir}/${1}/requirements.txt" --target=/asset-output
+    pip install --requirement="${script_dir}/${1}/requirements.txt" --target=${asset_root}
```

Comment:
`-target=${asset_root}` should be quoted now that there's a variable in that string.

Reply:
Since 49287826c0b951828565effd6d92cb237f139e08 we use single Lambda bundling script